pygeo/mphys/mphys_dvgeo.py

```diff
@@ -589,6 +589,25 @@ def nom_addVolumeConstraint(self, name, leList, teList, nSpan=10, nChord=10, sca
         )
         self.add_output(name, distributed=False, val=1.0)
 
+    def nom_addSurfaceAreaConstraint(self, name, scaled=True, surface_name="default"):
```

Comment:
Can you add the rest of the parameters besides bounds and scale?

Reply:
Done, added them to `addVolumeConstraint` and `addProjectedAreaCosntraint` too